Makefile

```diff
@@ -22,7 +22,7 @@ docs:
 test:
 	@flake8
 	@isort --check-only --diff formtools tests
-	@ coverage run `which django-admin` test tests
+	@ python -W error::DeprecationWarning -W error::PendingDeprecationWarning -m coverage run `which django-admin` test tests
```

Comment:
I use this pattern on all my projects to expose all warnings - even those from coverage or Django.